### DISCUSSION TITLE
any way trino can extract data from MySQL column that was compressed?

### DISCUSSION THREAD

**Discussion by smetis** [trinodb/trino#8717] [Q&A] [answered]
Hi,
I have mysql data that utilized MySQL compress function on large json string. I wonder if there is any way Trino can uncompress the data and perform SQL on it?
https://dev.mysql.com/doc/refman/8.0/en/encryption-functions.html

thanks.

**Comment by ebyhr** [comment, marked as answer]
There is no such feature in Trino side as far as I know. As workaround, we can prepare a MySQL view having columns uncompressing the data.

**Reply by smetis** [reply]
Thanks for the response. I will look into view option. Also, do you happen to know if redash will support Trino in near future? I only see presto in redash datasource list as of now.

**Reply by ebyhr** [reply]
The PR was already merged (https://github.com/getredash/redash/pull/5411). I'm not sure about the release status though. 
cc: @dungdm93

**Reply by smetis** [reply]
cool, will keep an eye on this PR. Thanks!
